src/test/java/seedu/task/logic/EditCommandTest.java

```diff
@@ -27,11 +27,11 @@ public void execute_editFloatTask_duplicate() throws Exception {
         model.addTask(helper.computingFloatTask());
         model.addTask(helper.computingEditedNameFloatTask());
         Task toBeEdited = helper.computingFloatTask();
-        model.editTask(toBeEdited, 0);
+        model.editTask(toBeEdited, toBeAdded);
```

Comment:
Sry, I don't understand this part. How come you are editing the task here and during the assertEditTaskCommandBehavior function? When you execute the logic in the "assert..." function, you are executing and making the changes to the model. I believe you should be editing the task in expectedAB instead? So far in your code, expectedAB's tasks have not been editted.